src/lib.ts

```diff
@@ -345,7 +348,7 @@ function haveCandies(a: Item, b: Item): boolean {
   return Array.from(candiesRequired.values()).every((val) => val === 1);
 }
 
-const rem = mainStatMaximizerString === $stat`Muscle` ? 2 : mainStatMaximizerString === $stat`Mysticality` ? 3 : 4;
+const rem = mainStatStr === $stat`Muscle` ? 2 : mainStatMaximizerString === $stat`Mysticality` ? 3 : 4;
```

Comment:
Again, we are comparing a `string` against a `$stat`

You should simply be exporting `const mainStat = myPrimestat()` and comparing if `mainStat === $stat'Muscle'`, and not `mainStatStr === $stat'Muscle'`

I'm not sure if there's a case where `mainStatStr` (i.e. "Muscle", "Mysticality", "Moxie") needs to exist, since you already have `mainStatMaximizerString` (i.e. "mus", "myst", "mox"). You only need to add `mainStat` (i.e. `$stat'Muscle'`, `$stat'Mysticality'`, `$stat'Moxie'`)

Reply:
So it looks like all the places I'm actually using this, a string is sufficient. It's CLIExecute, primarily. I made a few changes just to try and be consistent, however.